### DISCUSSION TITLE
Champion Power Equipment Axis Transfer Switch

### DISCUSSION THREAD

**Discussion by jspanitz** [jgyates/genmon#1144] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have submitted logs

### What is/are your question(s)?

Is there any way to support this switch or has it been looked at before?

**Comment by jgyates** [comment]
This switch has not been looked at before. I do not know if it supports modbus as a google search did not turn up anything useful. Do you know if this switch supports modbus? What generator are you using, that is relevant as the serial modbus is often routed to the generator controller. If you are using a generac residential then you may be using the ATS with the generator controller in two wire mode, which genmon supports.
